packages/node-http-handler/src/node-http-handler.ts

```diff
@@ -71,7 +76,7 @@ export class NodeHttpHandler implements HttpHandler {
   }
 
   private resolveDefaultConfig(options?: NodeHttpHandlerOptions | void): ResolvedNodeHttpHandlerConfig {
-    const { requestTimeout, connectionTimeout, socketTimeout, httpAgent, httpsAgent } = options || {};
+    const { requestTimeout, connectionTimeout, socketTimeout, httpAgent, httpsAgent} = options || {};
```

Comment:
Prettier is still not run on this. Either the prettier plugin is not installed in VSCode, or precommit hook is not being run. Can you manually run prettier?

```console
$ yarn prettier --write packages/node-http-handler/src/node-http-handler.ts
```